src/components/Adds/Adds.js

```diff
@@ -0,0 +1,112 @@
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+
+export default function Adds() {
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const response = await fetch('http://127.0.0.1:3000/cars/create', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        model: e.target.model.value,
+        color: e.target.color.value,
+        image: e.target.image.value,
+        range: e.target.range.value,
```

Comment:
Good work on implementing most of the required fields. However, you left out the description which is also essential.

![image](https://user-images.githubusercontent.com/59999191/197911164-a9a5b23a-64b7-46c3-b48d-0b3e01dae816.png)

Reply:
Done